diplom.py

```diff
@@ -52,33 +61,31 @@ def mean_average_precision(predicted_probability_distribution,
 # print train_X[0], train_y[0]
 # print test_X[0], test_y[0]
 
-from sklearn import linear_model
-logistic = linear_model.LogisticRegression(C = 1, solver = 'lbfgs', 
-    multi_class = 'multinomial', max_iter = 100)
-print('LogisticRegression score: %f'
-      % logistic.fit(train_X, train_y).score(test_X, test_y))
+
+if args.model == "logistic_regression":
+    logistic = linear_model.LogisticRegression(C = 1, solver = 'lbfgs', 
+                                               multi_class = 'multinomial', max_iter = 100)
```

Comment:
Also, please remove spaces when passing arguments: "C = 1" => "C=1" like you did in other places so far.